### DISCUSSION TITLE
Action Save row : value relationship field into another column

### DISCUSSION THREAD

**Discussion by JeroenP1974** [Budibase/budibase#14624] [Help] [answered]
I could us some help with this problem.....I have two mysql tables. Products and Groupnumber. 
Both tables are related to each other with an relationship (groupnr). If I want to add a new product I would like to use the value of the chosen realtionshipfield from a form (used to add product) als be written/used in another column of the same table. When I select the form field (in my case : {{ New Form.Fields.GroepNr }}) the id of the link will be written in the field groepnummer but its a numeric field and a error/warning is occurd. I als tried something with primaryDisplay but no luck. No error is occurd but the data is not written in the table column. Thank you for all the help and suggestions.

![Schermafbeelding 2024-09-22 171240](https://github.com/user-attachments/assets/8d1f588d-44b6-4117-b9bd-9c9da7381995)

**Comment by mikesealey** [comment, marked as answer]
Hey @JeroenP1974 We have a guide on roughly this scenario in the documentation you can look at it [here](https://docs.budibase.com/docs/save-two-linked-tables-in-one-form)

tl;dr add another save row action with and in the relationship column add the _id of the saved row from the step before.

Hope this helps!
